### PR TITLE
fix: add data-test-id attribute for testing

### PR DESCRIPTION
**Pull request by sammeyerreed**

### CODE REVIEW

File: lti-launch.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ class LtiLaunch extends LitElement {
 	render() {
 		const iFrameClasses = { 'd2l-content-frame-default-width': !this.iFrameWidth, 'no-border': true };
 		return html`<div class="d2l-content-frame">
-		<iframe id="lti-launch-id" class="${classMap(iFrameClasses)}" allow="microphone *; camera *; autoplay *; display-capture *"
+		<iframe id="lti-launch-id" data-test-id="lti-launch-id" class="${classMap(iFrameClasses)}" allow="microphone *; camera *; autoplay *; display-capture *"
```
Comment (sammeyerreed) [inline lti-launch.js:59]: @richmar10 is this what you're expecting?

Comment (richmar10) [reply lti-launch.js:59]: awesome...thx!

Comment (richmar10) [reply lti-launch.js:59]: what about something more descriptive like `data-test-id="lti-launch-frame"`?    It'll make the selector read a little nicer.

Comment (sammeyerreed) [reply lti-launch.js:59]: Yeah sounds good

**Review by maggiesazio — APPROVED** [review]

**Review by alogan519 — APPROVED** [review]

**Pull request merged**